Update little bit

MhankBarBar opened the pull request:
This update allows you to send sticker message and quoted message

Example quoted message
```python
from wamd.messages import ExtendedTextMessage
msg = ExtendedTextMessage(to=message["from"], text="Who?", quoted=message)
client.relayMessage(msg)
```

Example sticker message
```python
from wamd.messages import StickerMessage
msg = StickerMessage(to=message["from"], url="path/to/file.webp")
client.relayMessage(msg)

TriggerTG committed:
Update ExtendedTextMessage and StickerMessage

TriggerTG committed:
Add example quoted message

MhankBarBar committed:
Whoops, my bad

krypton-byte committed:
simple download MediaMessage, custom sticker metadata

MhankBarBar committed:
Merge pull request #1 from krypton-byte/main

simple download MediaMessage, custom sticker metadata

MhankBarBar committed:
Add option in media message

Serenity-77 commented:
when accessing message instance attributes you can directly use 
`message['something]`  instead of directly using `message._attrs`
it will return `None`  if key not found

krypton-byte committed:
downloadAndSave

MhankBarBar commented:
I use `message._attrs` to be able to call the `.get` function, if using the `.get` function, if the key doesn't exist it can be set to `False` or something else, not just `None`

krypton-byte committed:
ContactMessage

krypton-byte committed:
ContactMessage(quoted)

Serenity-77 commented:
> I use `message._attrs` to be able to call the `.get` function, if using the `.get` function, if the key doesn't exist it can be set to `False` or something else, not just `None`

Serenity-77 commented:
oke oke

MhankBarBar commented:
so? should i create a new function in the class `WhatsappMessage` to access `_attrs` and to keep `_attrs` private?
Like this
```python
class WhatsappMessage:
    def __init__(self, **kwargs):
        self._attrs = {}

    @property
    def getAttrs(self):
        return self._attrs
```

Serenity-77 commented:
> so? should i create a new function in the class `WhatsappMessage` to access `_attrs` and to keep `_attrs` private? Like this
> 
> ```python
> class WhatsappMessage:
>     def __init__(self, **kwargs):
>         self._attrs = {}
> 
>     @property
>     def getAttrs(self):
>         return self._attrs
> ```
> 
> But wait, private? If private should add another `_`, like this for example:
> 
> ```python
> class WhatsappMessage:
>     def __init__(self, **kwargs):
>         self.__attrs = {}
> 
>     @property
>     def getAttrs(self):
>         return self.__attrs
> ```

```shell
def __getitem__(self, item):
        try:
            return self._attrs[item]
        except KeyError:
            return None
```

is there any diffeerence between `message._attrs.get("something", "default")` and `"default" if  message['something'] is None else message['something']`?

but anyway just do what you want if it makes you comfortable

MhankBarBar commented:
It's simpler to use the get function instead of the ternary operator

MhankBarBar committed:
add list, location, and live location message

krypton-byte committed:
Update messages.py

krypton-byte committed:
ContactsArrayMessage && fix typo

krypton-byte committed:
Merge branch 'MhankBarBar-main'

MhankBarBar committed:
Merge pull request #2 from krypton-byte/main

Added ContactMessage, ContactsArrayMessage And downloadAndSave

krypton-byte committed:
filetype

MhankBarBar committed:
buttons message

krypton-byte committed:
Merge branch 'MhankBarBar:main' into main

krypton-byte committed:
productMessage

krypton-byte committed:
- added Product Message
- added custom header like image, video, location, document and text in button Message
Merge branch 'main' of https://github.com/krypton-byte/wamd

MhankBarBar committed:
Merge pull request #3 from krypton-byte/main

added ProductMessage & custom header(buttonMessage)

krypton-byte committed:
Update constants.py

krypton-byte committed:
feat: getProfilePic etc

MhankBarBar committed:
Merge pull request #4 from krypton-byte/main

getProfilePic
